command/list.go

```diff
@@ -46,9 +46,9 @@ func (c *ListCommand) Run(args []string) int {
 			instanceName = ""
 
 			for _, tag := range instance.Tags {
-				if *tag.Key == "Name" {
-					instanceName = *tag.Value
-					break
+				instanceName = instanceName + *tag.Key
+				if len(*tag.Value) > 0 {
+					instanceName = instanceName + "=" + *tag.Value + ","
```

Comment:
I think this is not good, because the end of string is always comma.

```
Name=name,Environment=prod,
```

So, how about this:

```go
ss := []string{}

for _, tag := range instance.Tags {
	s := *tag.Key

	if len(*tag.Value) > 0 {
		s += "=" + *tag.Value
	}

	ss := append(ss, s)
}

strings.Join(ss, ",")
```